sites/footers/reports/a11y-1.json

```diff
@@ -1,902 +1,2029 @@
-      "earl:test": "WCAG20:audio-description-or-media-alternative-prerecorded",
-      "outcome": "Pass",
+      "@id": "_:sc_1.2.3",
+      "earl:test": "WCAG21:audio-description-or-media-alternative-prerecorded",
+      "earl:subject": "_:subject",
       "earl:outcome": "earl:passed",
-      "notes": "",
-      "techniques": [
-        "WCAG21Tech:general/G69",
-        "WCAG21Tech:general/G68",
-        "WCAG21Tech:general/G78",
-        "WCAG21Tech:general/G173",
-        "WCAG21Tech:general/G8",
-        "WCAG21Tech:general/G203"
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G69",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G68",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G78",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G173",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G8",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G203",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        }
       ],
       "earl:mode": "earl:automatic",
       "@type": ["earl:TestResult", "earl:Assertion"]
     },
     {
-      "label": "SC 1.2.4 Captions (Live) (Level AA)",
-      "earl:test": "WCAG20:captions-live",
-      "outcome": "Pass",
+      "@id": "_:sc_1.2.4",
+      "earl:test": "WCAG21:captions-live",
+      "earl:subject": "_:subject",
       "earl:outcome": "earl:passed",
-      "notes": "",
-      "techniques": [
-        "WCAG21Tech:general/G9",
-        "WCAG21Tech:general/G93",
-        "WCAG21Tech:general/G87"
+      "dct:description": "",
+      "earl:result": [
+        {
+            "earl:test": "WCAG2xTech:general/G9",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G93",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        },
+        {
+            "earl:test": "WCAG2xTech:general/G87",
+            "earl:outcome": "earl:cantTell",
+            "earl:subject": "_:subject"
+        }
       ],
       "earl:mode": "earl:automatic",
       "@type": ["earl:TestResult", "earl:Assertion"]
     },
     {
-      "label": "SC 1.2.5 Audio Description (Prerecorded) (Level AA)",
-      "earl:test": "WCAG20:captions-live",
-      "outcome": "Pass",
+      "@id": "_:sc_1.2.5",
+      "earl:test": "WCAG21:captions-live",
```

Comment:
This "earl:test" at line 244 should be for audio-description instead of captions-live. I believe this was a mistake in my initial PR.

Reply:
Fixed